src/sections/2021/section-3.js

```diff
@@ -0,0 +1,461 @@
+          keyMap={['']}
+          dataPoints={[
+            ['Adoption (developer, designer, etc.)', [42]],
+            ['Contribution', [37]],
+            ['Product road map', [36]],
+            ['Overcoming technical/creative debt', [35]],
+            ['Project management or process', [29]],
+            ['Governance', [23]],
+            ['Internal education', [21]],
+            ['Proving Value', [21]],
+            ['Staffing', [21]],
+            ['Funding', [10]],
+            ['Finding an executive champion', [7]],
+            ['Subscriber engagement', [4]],
+            ['Other (please specify)', [6]]
+          ]}
+        />
+      </Figure>
+    </GridCell>
+
+    <GridCell>
+    <Figure count="3.2" caption="Responses: 158 | Respondents were asked to select all that apply.">
+      <BarChart
+        headingLevel="h3"
+        title={<>What are the top <em>challenges</em> your design system team is facing at this moment</>}
+        keyMap={['']}
+        dataPoints={[
+          ['Overcoming technical/creative debt', [47]],
+          ['Contribution', [45]],
+          ['Adoption', [44]],
+          ['Staffing', [39]],
+          ['Internal education', [36]],
+          ['Product Road Map', [35]],
+          ['Governance', [34]],
+          ['Project management or process', [32]],
+          ['Providing value', [26]],
+          ['Funding', [23]],
+          ['Finding an executive champion', [16]],
+          ['Subscriber engagement', [6]],
+          ['Other (please specify)', [7]]
+        ]}
+      />
+    </Figure>
+    </GridCell>
+
+    <GridCell>
+      <Figure count="3.3">
+        <h2 class="cmp-type-h3">Top priorities and challenges</h2>
+        <p>Across both top priorities and challenges, three areas stood out, which we will explore more deeply in the next sections.</p>
+        <h3>1</h3>
+        <p>Encouraging Adoption</p>
+        <h3>2</h3>
+        <p>Engaging Contributors</p>
+        <h3>3</h3>
+        <p>Overcoming Debt</p>
+      </Figure>
+    </GridCell>
+
+    <GridCell>
+      <h2 class="cmp-type-h3">1 - Encouraging Adoption</h2>
+    </GridCell>
+
+    <GridCell>
+      <Figure count="3.4">
+      <h3> Priority #1 </h3>
+      <p>42% of in-house respondents selected increasing adoption as a priority</p>
+
+      <h3>Challenge #3</h3>
+      <p>44% of in-house respondents selected increasing adoption as a challenge</p>
+      </Figure>
+    </GridCell>
+
+    <GridCell>
+      <h2 class="cmp-type-h3">Adoption continues to be a focus for many teams</h2>
+      <p>Adoption difficulties have been high on respondents’ lists of concerns since our 2018 Design Systems Survey and remain there in 2021.</p>
+    </GridCell>
+
+    <GridCell>
+      <Quote styleFormat="small"
+      cite="-Individual contributor, moderately successful design system">
+        <p>When used well it has had a large uplift in sales and performance. However, teams still need to be pushed to use it properly.</p>
```

Comment:
Anything part of a quote that's bold in the doc should be placed in `<em><strong>` here to make it bold and red. Sorry, should have mentioned that earlier.

So here "still need to be pushed" should be bold and red.